taskvine/src/manager/vine_current_transfers.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (C) 2022- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#include "vine_worker_info.h"
+#include "uuid.h"
+
+#define VINE_FILE_SOURCE_MAX_TRANSFERS 1
+
+struct vine_transfer_pair {
+    struct vine_worker_info *to;
+    char   *source;
+};
+
+char *vine_current_transfers_add(struct vine_manager *q,struct vine_worker_info *to, char *source);
```

Comment:
Most string parameters to functions should be `const char *` since you aren't going to modify the contents of those strings.